### PR TITLE
Current theming

### PR DESCRIPTION
**Pull request by bry-beep**

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules/
```
Comment (cschucode) [inline client/.gitignore:1]: Do we need multiple .gitignore files for each directory? Couldn't we have one gitignore file in the root directory that also references the sub-directories? https://git-scm.com/docs/gitignore

File: client/src/Theme.jsx [hunk 2]
```diff
@@ -1,21 +1,81 @@
...
-  colors: {
-    powderWhite: '#FFFDF9',
-    persianGreen: '#06B49A',
-    lightBlue: '#AFDBD2',
-    onyx: '#36313D',
+export const GlobalStyle = createGlobalStyle`
+  body {
+  background: white;
+  color: black;
+  /* background-color: ${({ theme }) => theme.color.purple}; */
```
Comment (cschucode) [inline client/src/Theme.jsx:8]: Are we planning on using the commented-out code? We would want to remove what we don't use before merging to main.

File: client/src/components/Blog.jsx [hunk 3]
```diff
@@ -4,17 +4,17 @@ import { connect } from 'react-redux'
 export const Blog = (props) => {
```
Comment (cschucode) [inline client/src/components/Blog.jsx:4]: Should this file be in this PR?

File: jobs/package.json [hunk 4]
```diff
@@ -4,9 +4,25 @@
...
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon index.js"
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+<<<<<<< HEAD
```
Comment (cschucode) [inline jobs/package.json:12]: Looks like a merge conflict here that will need to be resolved here.

File: client/.gitignore (hunk 1 above)
Comment (bry-beep) [reply client/.gitignore]: Yes! Was outdated but now fixed with cache removed as well

File: client/src/Theme.jsx (hunk 2 above)
Comment (bry-beep) [reply client/src/Theme.jsx]: Some, yes - leaving as an option. But removed most of it/ what we don't need reference to

**Comment by Kodylow** [comment]
copied over into main and merged

**Pull request closed without merging**